spec/web_helpers.rb

```diff
@@ -0,0 +1,43 @@
+module User_helpers
+  def sign_up(email: "test@example.com",
+              password: "testtest",
+              password_confirmation: "testtest")
+    visit('/')
+    click_link('Sign up')
+    fill_in(:user_email, with: email)
+    fill_in(:user_password, with: password)
+    fill_in(:user_password_confirmation, with: password_confirmation)
+    click_button('Sign up')
+  end
+
+  def create_post(description)
+    visit("/posts")
+    click_link("Create Post")
+    attach_file("post[image]", File.expand_path("./spec/test.jpg"))
+    fill_in("Description", with: description)
+    click_button("Create Post")
+  end
+
+  def edit_post(description)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `description`. If it's necessary, use `_` or `_description` as an argument name to indicate that it won't be used. You can also write as `edit_post(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)